index.d.ts

```diff
@@ -4,12 +4,7 @@
 /**
  * Options pass to Registry.metrics()
  */
-export interface MetricsOpts {
-	/**
-	 * Whether to include timestamps in the output, defaults to true
-	 */
-	timestamps?: boolean;
-}
+export interface MetricsOpts {}
```

Comment:
I left it because it doesn't do any harm to have an empty options object, and it'll be there in the future if there ever is an option. But then, YAGNI, so I'll remove it.